frontend-calculator/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
I think that you include some files and folders in this gitignore, such as env files and auto generated folders. You can see the latest version of gitignore [here](https://github.com/OsmanRodrigues/rollups-examples/blob/dev/frontend-m2cgen/.gitignore).